scheduler/build.gradle

```diff
@@ -23,23 +22,24 @@ sourceSets {
   }
 }
 
-def nameMerlinSightJar = "merlinsight-rules.jar"
-def testResourceDir = "src/test/resources"
-task merlinsightjar(type: Jar, dependsOn: compileTestJava)  {
-    from sourceSets.test.output
-    include 'gov/nasa/jpl/aerie/scheduler/MerlInsightRules.class'
-    archiveFileName = nameMerlinSightJar
+def nameMerlinSightJar = 'merlinsight-rules.jar'
+def testResourceDir = 'src/test/resources'
+
+task merlinsightJar(type: Jar, dependsOn: compileTestJava)  {
+  from sourceSets.test.output
+  include 'gov/nasa/jpl/aerie/scheduler/MerlInsightRules.class'
+  archiveFileName = nameMerlinSightJar
```

Comment:
Just a note, no semantic changes were made in this file. I cleaned up some formatting, quotes, and variable names.